src/main/java/TaskList.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+
+/**
+ * Manages an ArrayList of Tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
+
+    public TaskList(){
+        this.taskList = new ArrayList<Task>();
+    }
+
+    /**
+     * Adds a new string as ToDo to the ArrayList and prints the Task added.
+     * @param taskName
+     */
+    public void addToDo(String taskName){
+        Task newToDo = new ToDo(taskName);
+        taskList.add(newToDo);
+        System.out.println("Added:" + System.lineSeparator() + "   " + newToDo);
+    }
+
+    /**
+     * Adds a new string as Deadline to the ArrayList and prints the Task added.
+     * @param taskName
+     * @param deadline
+     */
+    public void addDeadline(String taskName, String deadline){
+        Task newDeadline = new Deadline(taskName, deadline);
+        taskList.add(newDeadline);
+        System.out.println("Added:" + System.lineSeparator() + "   " + newDeadline);
+    }
+
+    /**
+     * Adds a new string as Event to the ArrayList and prints the Task added.
+     * @param taskName
+     * @param eventTime
+     */
```

Comment:
Appreciate the use of comments to explain the code, well done!

Reply:
Thank you!